### PR TITLE
chore: change name to reflect branding

### PR DESCRIPTION
**Pull request by kcoderhtml**
# Name Change
## Description:

Change the name of the org across the site and add a proper README

## Linked Issues:
Closes: #20

### CODE REVIEW

**Comment by jdogcoder** [comment]
Looks good so far, lmk when it's ready for me!

**Comment by kcoderhtml** [comment]
All good on my side!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,15 @@
-
-# site
+# Purple Bubble Website
```
Comment (jdogcoder) [inline README.md:1]: This should be `Purple Bubble's Website` I think :)

**Review by jdogcoder — CHANGES_REQUESTED** [review]

**Comment by jdogcoder** [comment]
Very minor change requested @kcoderhtml

**Review by jdogcoder — APPROVED** [review]

**Pull request merged**